packages/shared/src/components/modals/AccountDetailsModal.module.css

```diff
@@ -8,3 +8,14 @@
     }
   }
 }
+.myFeedModal {
+& :global(.modal) {
+    padding: 3rem 0 4.0625rem 0;
+
+@screen responsiveModalBreakpoint {
+  max-height: 40rem;
+  padding-top: 0;
+  padding-bottom: 0;
+}
+}
+}
```

Comment:
Something looks odd with indentation here

Reply:
Fixed, for some reason modules never reflect 😅